cicecore/drivers/nuopc/cmeps/ice_import_export.F90

```diff
@@ -481,117 +502,161 @@ subroutine ice_import( importState, rc )
-    ! Note also that the read in below includes only values needed
-    ! by the thermodynamic component of CICE.  Variables uocn, vocn,
-    ! ss_tltx, and ss_tlty are excluded. Also, because the SOM and
-    ! DOM don't  compute SSS.   SSS is not read in and is left at
-    ! the initilized value (see ice_flux.F init_coupler_flux) of
-    ! 34 ppt
-
     ! Use aflds to gather the halo updates of multiple fields
     ! Need to separate the scalar from the vector halo updates
 
     allocate(aflds(nx_block,ny_block,nflds,nblocks))
     aflds = c0
-
-    ! import ocean states
-
+    ! +------------------------------------------------------------------+
+    ! |                 Thremodynamic import variables                   |
+    ! +------------------------------------------------------------------+
+    !
+    ! In the following section all vairables need to run thermodynamic 
+    ! are imported. 
+    !
+    ! Dynamic Quantites are imported bellow (Search for Dynamics) 
+    !
+    !                   +---------------------------+
+    !                   |       Ocean Import        |
+    !                   +---------------------------+
+
+    ! Sea surface temperature --------------------------------------------
     call state_getimport(importState, 'So_t', output=aflds, index=1, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
+    ! Sea surface salinity ------------------------------------------------
     call state_getimport(importState, 'So_s', output=aflds, index=2, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
-    ! import atm states
+    ! Deep ocean heat flux at base of mixed layer -------------------------
 
+    !call state_getimport(importState, 'Fioo_q', output=aflds, index=9, &
+    !     areacor=med2mod_areacor, rc=rc)
+    !if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    !call ESMF_LogWrite("Ocean/ice flux imported! ", ESMF_LOGMSG_INFO)
+    ! import atm fluxes
+
+        
+    !                   +---------------------------+
+    !                   |       Atmos Import        |
+    !                   +---------------------------+
+    
+    ! Lowest atmospheric height -------------------------------------------
     call state_getimport(importState, 'Sa_z', output=aflds, index=3, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
+    ! Calc/Imp surface pressure -------------------------------------------------------------
     if (State_FldChk(importState, 'Sa_ptem') .and. State_fldchk(importState, 'Sa_dens')) then
        call state_getimport(importState, 'Sa_ptem', output=aflds, index=4, rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
        call state_getimport(importState, 'Sa_dens', output=aflds, index=5, rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
-    else if (State_FldChk(importState, 'Sa_pbot')) then
-       call state_getimport(importState, 'Sa_pbot', output=aflds, index=6, rc=rc)
+    else if (State_FldChk(importState, 'Sa_pslv')) then
```

Comment:
@SmithJos13 Again, we could not change variable names. If it is really necessary, we could introduce new ones. But, again there could be another solution by changing CDEPS configuration.